velox/functions/prestosql/aggregates/PrestoHasher.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/vector/DecodedVector.h"
+
+namespace facebook::velox::aggregate {
+
+class PrestoHasher {
+ public:
+  explicit PrestoHasher(
+      const BaseVector& vector,
+      const SelectivityVector& rows) {
+    vector_ = std::make_shared<DecodedVector>(vector, rows);
+    createChildren();
+  }
+
+  /// Function that computes a Presto equivalent hash.
```

Comment:
Drop "Function that" for clarity. "Computes ....".

Would readers know what Presto equivalent hash is? Perhaps, clarify , "Computes a hash using algorithm identical to the one used by Presto checksum() aggregate function. This algorithm is based on xxhash64."